src/main/java/tutoraid/commons/core/HelpGuide.java

```diff
@@ -136,10 +154,12 @@
             + "command that changes the data. There is no need to save manually.";
 
     public static final String EDIT_DATA_TITLE = "Editing the data file";
-    public static final String EDIT_DATA_DESC1 = "TutorAid data are saved as a JSON file ";
-    public static final String EDIT_FILEPATH = "[JAR file location]/data/tutoraid.json";
-    public static final String EDIT_DATA_DESC2 = "Advanced users are welcome to update data directly by editing "
-            + "that data file.";
+    public static final String EDIT_DATA_DESC1 = "TutorAid student data are saved as a JSON file ";
+    public static final String EDIT_STUDENTS_FILEPATH = "[JAR file location]/data/tutorAidStudents.json";
+    public static final String EDIT_DATA_DESC2 = ", while TutorAid lesson data are saved as a JSON file ";
+    public static final String EDIT_LESSONS_FILEPATH = "[JAR file location]/data/tutorAidLessons.json";
+    public static final String EDIT_DATA_DESC3 = "Advanced users are welcome to update data directly by editing "
+            + "any of the two data files.";
     public static final String EDIT_CAUTION = "Caution: If your changes to the data file makes its format invalid, "
```

Comment:
Maybe can update to indicate that invalid student data -> student data discarded rather than saying all of it will be gone

Reply:
Updated to indicate that invalid format or data will render that data file invalid and be replaced with an empty data file